src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -20,6 +22,12 @@
 
 using LiteralOrIri = ad_utility::triple_component::LiteralOrIri;
 
+constexpr char ExceptionMessagePrefix[] =
```

Comment:
can maybe be `std::string_view` and either `static` or in the anonymous namespace.